Module 10 Exercise Part 2

taylorglass16 opened the pull request:
I fixed the RMSE values by switching test_data2 to train_data2. I added my part 2 contributions with the bootstrapping method and appropriate graphs.

rachelr989 commented:
> I fixed the RMSE values by switching test_data2 to train_data2. I added my part 2 contributions with the bootstrapping method and appropriate graphs.

Thank you, Taylor. I was unclear about the RMSE values. May I ask why we ask the formula to refer to the training data when comparing the predictions rather than the testing data?

taylorglass16 commented:
I sent you a DM on discord because I think I will see those messages
before I see my email!

On Thu, Mar 21, 2024 at 11:22 AM rachelr989 ***@***.***>
wrote:

> I fixed the RMSE values by switching test_data2 to train_data2. I added my
> part 2 contributions with the bootstrapping method and appropriate graphs.
>
> Thank you, Taylor. I was unclear about the RMSE values. May I ask why we
> ask the formula to refer to the training data when comparing the
> predictions rather than the testing data?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rachelr989/MADAcourserepo/pull/4#issuecomment-2012616181>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDFVQ4TCKM5LHUOFKIHIKK3YZL3L7AVCNFSM6AAAAABFBUNBECVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDAMJSGYYTMMJYGE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
